earth_enterprise/src/third_party/gdal/SConscript

```diff
@@ -74,7 +80,18 @@ gdal_extract = gdal_env.Command(
         'tar xzf %s\n'
         'touch %s'% (current_dir, current_dir, gdal_source, gdal_target))])
 
-# [2] Extract Kakadu
+
+# [2] Extract openjpeg
+gdal_target = '%s/.extract_openjpeg' % current_dir
+openjpeg_extract = gdal_env.Command(
+    gdal_target, [openjpeg_source, gdal_extract],
+    [gdal_env.MultiCommand(
+        'cd %s\n'
+        'tar xzf %s\n'
+        'touch %s' % (build_root, openjpeg_source, gdal_target))])
+
+
+# [3] Extract Kakadu
```

Comment:
If your comment is about kakakdu, then I believe we intentionally left it there, so whoever wants to support in their environment, it should be easy to un-comment and use it. 
It serves like config settings in apache or postgress sort of, where not everyone uses everything so its commented out and they can uncomment as required.

On the other hand , I would say we can possibly cleanup and make different scons constructs for kakadu and Mr SID and just have few lines here to include them, but that is not scope of openjpeg work.